dev-lang/idris/idris-1.3.3.ebuild

```diff
@@ -68,6 +68,9 @@ src_prepare() {
 	# runs dist/build/idris directly and breaks sandboxing
 	export LD_LIBRARY_PATH="${S}/dist/build${LD_LIBRARY_PATH+:}${LD_LIBRARY_PATH}"
 
+	# Upstream did this in commit cae9aa29
+	cabal_changedeps 'Cabal >= 2.4 && < 3.1,' 'Cabal >= 2.4 && <= 3.4'
```

Comment:
You probably meant `cabal_chdeps`. `cabal_changedeps` does not exist, which might mean the change did nothing.

Reply:
Whoops, I did mean this!

I'm curious why there wasn't an error during build though, if I tried to call a command that doesn't exist.

Reply:
`bash` or `portage` does not fail ebuild for calling non-existent programs. But there should be a QA warning after package merge.

Reply:
Hrm, I must have forgotten to run repoman on this one.

As I stated in a comment at the bottom, I've restarted this PR in a new
branch. I'm trying to be more systematic about it, including running
repoman and pkgcheck for each new or changed ebuild.